website/src/pages/documents/document.page.tsx

```diff
@@ -82,6 +83,10 @@ const TabSet = ({ doc }: { doc: Document }) => {
           </Tab>
         </TabList>
       </div>
+      <div id="scroll-top" className={css.scrollTop} onClick={() => window.scrollTo({top: 0, behavior: 'smooth'})}>
+      <h3><RiArrowUpCircleFill size={55}/></h3>
```

Comment:
_Please, keep your HTML semantic!_ [Quoting MDN](https://developer.mozilla.org/en-US/docs/Glossary/Semantics): "HTML should be coded to represent the data that will be populated and not based on its default presentation styling." In this case, that means you should **not** use a heading element here since this button does not function as a heading. An anchor link could work, or maybe a non-semantic element like div.